lib/service/src/main/java/io/atlasmap/service/AtlasService.java

```diff
@@ -78,10 +82,15 @@
 import io.swagger.annotations.ApiResponse;
 import io.swagger.annotations.ApiResponses;
 
+import static io.atlasmap.v2.MappingFileType.ADM;
+import static io.atlasmap.v2.MappingFileType.GZ;
+
 @Api
 @Path("/")
 public class AtlasService {
 
+    static final String MAPPING_NAME_PREFIX = "UI.";
+    static final String MAPPING_NAME_SUFFIX = ".default";
```

Comment:
Do we need this `.default` suffix?

Reply:
Initially i used it to distinguish in mappings/?filter API between UI.1 and UI.11 (where all mapping files were in the same directory) without changing the API. It can be removed now